applications/FluidDynamicsApplication/custom_elements/embedded_navier_stokes.h

```diff
@@ -183,8 +183,12 @@ class EmbeddedNavierStokes : public NavierStokes<TDim, TNumNodes>
             }
         }
 
+        if (rData.n_pos != 0 && rData.n_neg != 0) {
+            this->Set(TO_SPLIT, true);
```

Comment:
I'm a bit concerned with using this flag here... Wasn't it the one for adaptive refinement? If so, it can conflict quite badly...

In a related note, for the new elements in my PR I'm using an IsSplit() method I added to the data class (which does this same check). We need to discuss this and adopt the same way of deciding if the element is split.

Reply:
Hi @jcotela, @pooyan-dadvand and me discussed this past week. We both agree that to add a new flag or to simply check the amount of positive and negative number of nodes is better to avoid interactions with other utilities (e.g., adaptive refinement or distance computation).